docs/README.md

```diff
@@ -8,6 +17,8 @@ For a quick guide on getting started, see the top-level [README](../README.md#ge
 
 ### Adding ty to your project
 
+Your project must use git for source control.
```

Comment:
sooo... I'm not too sure about the exact details but I _think_ we would also look at a `.ignore` file even if your project doesn't use git for source control. @MichaReiser is our expert on configuring ty!

Reply:
Reworded; happy to reword again once I have clearer facts.